test/testdrive/basic.td

```diff
@@ -526,6 +526,27 @@ a b
 1 2
 2 3
 
+$ kafka-ingest format=raw topic=test-bytes timestamp=1
+test-string
+
+> CREATE SOURCE test_bytes
+  FROM KAFKA BROKER '${testdrive.kafka-addr}' TOPIC 'testdrive-test-bytes-${testdrive.seed}'
+  FORMAT BYTES;
+
+> CREATE MATERIALIZED VIEW test_bytes_view
+  AS SELECT * FROM test_bytes;
+
+# TODO@jldlaughlin: Change the testdrive implementation to not display a string here?
```

Comment:
To make what's happening here more explicit, you could do something similar to what we do in bytes.td: ingest a special unicode character using `kafka-ingest`, verify that you see UTF-8 encoded bytes when you select from the source directly, and then verify that you see the copyright symbol verbatim after a `convert_from` call.